toDoApp/src/navigation/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { createStackNavigator } from '@react-navigation/stack'
+import { Home } from '../screens/home'
+
+const Stack = createStackNavigator()
+
+export const AppContainer = () => (
+  <Stack.Navigator>
+    <Stack.Screen name="home" component={Home} />
```

Comment:
Maybe its better to put this "home" string inside an Enum called Routes or something to avoid repeat the string in some other places. Its better to enhance the reuse and the avoid unpleasant bugs.